docs/tutorials/platform_beginner/4_displaying_content_of_a_ride.md

```diff
@@ -89,6 +88,11 @@ The Starting Point and Ending Point Google Maps rendering is not yet done. You w
 
 We still haven't set any matching rule for our new Content Type Ride, so let's add one that will render a specific template for a Ride Content Type. 
 
+​    !!! note
+
+​    In every example, we show you a valid YAML excerpt. Keep the existing keys and add the missing content in the YAML file.
+​    YAML does not expect duplicate keys.
```

Comment:
Maybe "accept" or even "allow" instead of "expect"? I think duplicate keys can cause errors in YAML, so it would make sense to phrase it more strongly.

Reply:
Good call:
> duplicate mapping keys throw a ParseException


https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Yaml/CHANGELOG.md